torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -4110,6 +4520,51 @@ def rref_error():
 
         dist.barrier()
 
+    def _trainer_func(self, rref, sparse):
+        m = MyEmbeddingBagModel(sparse=sparse)
+        loss_fn = nn.MSELoss()
+        outputs = m(torch.rand(10, 10).long())
+        loss_fn(outputs, torch.rand(10, 10)).backward()
+        gradient = list(m.parameters())[0].grad
+        fut = rref.rpc_async().average(rref, gradient)
```

Comment:
this can be done in a followup PR. Let's also test using RPC to sync gradients in a DDP comm hook, as that's the expected use case.